My first Pull request for Hello Microverse Project

Miliyonayalew opened the pull request:
## Hello Microverse Project Milestone
In this Milestone I:

- Set up Github Action
- Configured (Set-up) linter for my local environment
- Installed packages for stylelint and hintrc
- Added index.html to the project
- Added styles.css to the project
- Added test.md to the project
- Created .gitignore file and added node_modules and test.md
- Added descriptive README.md file in professional way

Reem-lab reviewed (commented):
Hi @Miliyonayalew, 👋🏻👋🏻 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you don't have a descriptive pull request and your README is not customized as well you can get it from Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:

- [ ] I noticed that you didn't install the backages for stylelint and hintrc kindly add them then scroll down and install their packages :white_check_mark: to make them works well and don't forget to put node_modules after installing the packages inside the gitignore file
scroll down [here](https://github.com/microverseinc/linters-config/tree/master/html-css) you will see the steps please add it to your project :white_check_mark:
![image](https://user-images.githubusercontent.com/58553711/184869109-2fd349fa-063c-4d4e-a5a1-72ebf482bc4c.png)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Miliyonayalew commented:
## Hello Microverse Project Milestone
In this Milestone I:

- Set up Github Action
- Configured (Set-up) linter for my local environment
- Installed packages for stylelint and hintrc
- Added index.html to the project
- Added styles.css to the project
- Added test.md to the project
- Created .gitignore file and added node_modules and test.md
- Added descriptive README.md file in professional way

Bondok6 reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Miliyonayalew 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No linter errors ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

<img src="https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966" />


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
